src/HeaderBar.vala

```diff
@@ -78,9 +78,15 @@ public class Mail.HeaderBar : Gtk.HeaderBar {
         app_menu.popover = app_menu_popover;
         app_menu.tooltip_text = _("Menu");
 
+        var reply_button = new Gtk.Button.from_icon_name ("mail-reply-sender", Gtk.IconSize.LARGE_TOOLBAR);
+        reply_button.halign = Gtk.Align.START;
```

Comment:
I don't think it's necessary to add an halign  here since hexpand is false by default